docs/vectors/sealedbox_vectors.rst

```diff
@@ -0,0 +1,10 @@
+SealedBox reference vectors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Since, at the time of writing, we haven't been able to find
```

Comment:
`Here's my idea what this could look like`

Since libsodium's tests do not provide reference data for the SealedBox construction, the implementation is verified with C-driver, a utility program that produces custom test vectors by making specific calls to libsodium API. These test vectors can then be reproduced with input/output via PyNaCl's API.

1. Download, compile and verify the C-driver (source below) with following commands:
    `<Insert commands here, make sure the user doesn't have to struggle with C or installing libsodium.>`

    This command should print following official test vectors that prove C-driver is working properly: 
   `<List official test vectors that prove correctness of the C-driver and provide link to the source of vectors.>`

2. Produce SealedBox specific test vectors with C-driver by running it again, this time with following arguments:
    `<insert commands here.>`

3. Verify Python API correctly uses libsodium and replicates SealedBox specific test vectors by running `box_seal_from_api.py` (source below) with following commands:
    `<Insert commands here.>`

`NOTE: The third one could also be implemented like this:`

3. Compare the custom SealedBox specific test vectors with test vectors used in PyNaCl's unittests:
    `<Include a snippet of sealed_box_ref.txt containing test vectors and link to the file>`

### Sources

C-driver
    `<c-driver source>`

SealedBox
    `<box_seal_from_api.py content>`